Makefile

```diff
@@ -16,6 +16,7 @@ install:
 	install -c -m 444 snare.conf.example ${PREFIX}/share/examples/snare
 
 distrib:
+	git status
```

Comment:
Is this for the logs?

Reply:
Mea culpa -- I should have removed that! Fixed in https://github.com/softdevteam/snare/pull/38/commits/7c8cd889688b3d6e3d2bdf667cd8bb9cdbe74e32.